lib/rules/attribute-hyphenation.js

```diff
@@ -22,14 +22,40 @@ module.exports = {
     schema: [
       {
         enum: ['always', 'never']
+      },
+      {
+        type: 'object',
+        properties: {
+          'ignore': {
+            type: 'array',
+            items: {
+              allOf: [
+                { type: 'string' },
+                { not: { type: 'string', pattern: ':exit$' }},
+                { not: { type: 'string', pattern: '^\\s*$' }}
+              ]
+            },
+            uniqueItems: true,
+            additionalItems: false
+          }
+        },
+        additionalProperties: false
       }
     ]
   },
 
   create (context) {
     const sourceCode = context.getSourceCode()
-    const options = context.options[0]
-    const useHyphenated = options !== 'never'
+    const option = context.options[0]
+    const optionsPayload = context.options[1]
+    const useHyphenated = option !== 'never'
+    let ignoredAttributes = []
+
+    if (optionsPayload && optionsPayload.ignore) {
+      ignoredAttributes = optionsPayload.ignore
+    } else {
+      ignoredAttributes = ['data-', 'aria-', 'slot-scope']
```

Comment:
You can set ignoredAttributes above and remove the whole `else` clause.

Reply:
This makes sense, I have done this change.